packages/neos-ui/src/Containers/RightSideBar/Inspector/PropertyGroup/index.js

```diff
@@ -48,7 +48,7 @@ export default class PropertyGroup extends PureComponent {
                         const itemType = $get('type', item);
                         const label = $get('label', item) || '';
 
-                        if (itemType === 'editor') {
+                        if (itemType === 'editor' && $get('hidden', item) !== true) {
```

Comment:
hmm I'd prefer if it worked not only with editors but also with views.
Alternative, add before this line: 
```
if ($get('hidden', item)) {
    return null;
}
```

Reply:
All right, I'll take care of it.